src/main/java/com/exadel/discount/repository/VendoLocationrRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.exadel.discount.repository;
+
+import com.exadel.discount.entity.VendorLocation;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface VendoLocationrRepository extends JpaRepository<VendorLocation, Long> {
+}
```

Comment:
Mistake in naming: VendorLocationRepository
And also we should use extends JpaRepository<VendorLocation, UUID> not Long, in each repository

Reply:
+